### PR TITLE
Package library in a standard way

### PR DESCRIPTION
**Pull request by psafont**
To prepare the library to work normally with pip install we:
- define ragelib as a regular python package
- move to setup.py and requirements.txt to pyproject.toml and setup.cfg

We remove Pipfile because it's only useful for locking library versions, only that should only be done on non-library packages as these may cause conflicts otherwise. If there are any bounds these can be define in the dependencies

Signed-off-by: Pau Ruiz Safont <pau.safont@citrix.com>

### CODE REVIEW

**Comment by edwintorok** [comment]
the lockfile might be useful in the sense: this is the version it is known to work with, although the .toml file you use seems like a better way of managing that.
I think we can use tools like `poetry` to produce a lockfile out of it.

**Comment by psafont** [comment]
we can use the locked versions in ragebot, here it might make sense to define the known versions as the minimum bounds

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = ragelib
+description = A helper library for RAGE
+long_description = file: README.md
+long-description-content-type = text/markdown; charset=UTF-8
+author = Xenserver
+author_email = perf@xenserver.com
```
Comment (edwintorok) [inline setup.cfg:7]: I doubt this email goes anywhere other than /dev/null we can try to fix that up when we get new email addresses.

Comment (edwintorok) [reply setup.cfg:7]: Is  the author email field required?

Comment (psafont) [reply setup.cfg]: the previous email was callum's, which was going to /dev/null. In any case it seems to work fine without the author fields

**Comment by edwintorok** [comment]
The library previously used Selenium 3.9, and 4.x is not backwards compatible, but I've got a PR that would fix that. So merging this.

**Pull request merged**